CHANGES/6636.feature

```diff
@@ -0,0 +1 @@
+Add model resources to allow pulp import export handle pulp_container content units.
```

Comment:
I suggest capturing this feature in this feature list https://docs.pulpproject.org/pulp_container/#features  e.g. "Support disconnected and air-gaped environments with pulp import/export facility'